spec/asyncapi.md

```diff
@@ -84,20 +84,23 @@ It means that the [application](#definitionsApplication) allows [consumers](#def
 
 ## <a name="definitions"></a>Definitions
 
+#### <a name="definitionsServer"></a>Server
+A server MAY be a message broker that is capable of sending and/or receiving between a [producer](#definitionsProducer) and [consumer](#definitionsConsumer). A server MAY be a service with WebSocket API that enables message-drive communication between browser-to-server or server-to-server.
```

Comment:
```suggestion
A server MAY be a message broker that is capable of sending and/or receiving between a [producer](#definitionsProducer) and [consumer](#definitionsConsumer). A server MAY be a service with WebSocket API that enables message-driven communication between browser-to-server or server-to-server.
```

Reply:
@derberg How about the "MAY" caps word is it true that I need to make it caps? cc. @alequetzalli

Reply:
yeap, it should be caps